models/odm.py

```diff
@@ -41,3 +42,25 @@ def token(self):
         """
         res = odm_requests.get_token_auth(self.url, self.user, self.password)
         return res.json()['token']
+
```

Comment:
I have to admit, I'm going to have to trust you on this one. I can see that it manages the encryption of passwords, which presumably avoids having passwords in plaintext visible anywhere, but I don't understand the actual implementation. I'm going to have to read more about this.